aws/resource_aws_codepipeline_test.go

```diff
@@ -488,10 +592,29 @@ func testAccPreCheckAWSCodePipeline(t *testing.T) {
 	}
 }
 
+func testAccPreCheckAWSCodePipelineAlternateRegion(t *testing.T) {
+	// There isn't a way to get the alternate region provider at PreCheck time, so hardcode it
+	if testAccGetAlternateRegion() == "us-gov-east-1" {
```

Comment:
Not sure, but this might be better as a check on `testAccGetAlternateRegionPartition() == "aws-us-gov"` instead…

Reply:
Wow this is annoying (on the AWS side), I'm fine with either, to be honest.